resources/profiles/Creality/machine/Creality CR-10 Max.json

```diff
@@ -1,12 +1,12 @@
 {
     "type": "machine_model",
     "name": "Creality CR-10 Max",
-    "model_id": "Creality_CR-10_Max",
+    "model_id": "Creality-CR10-Max",
```

Comment:
let's keep the model_id same.
previously created profiles will run into troubles if we change model id

Reply:
Oh okay, I will just change it

Reply:
I have reset the Model_ID for all Creality and SecKit back to the original ID.